docs/sapling-comparison.md

```diff
@@ -25,22 +25,26 @@ Here is a list of some differences between jj and Sapling.
   is [automatically snapshotted by every command](working-copy.md). New files
   are automatically tracked and deleted files are automatically untracked. This
   has several advantages:
-    * The working copy is effectively backed up every time you run a command.
-    * No commands fail because you have changes in the working copy ("abort: 1
+
+  * The working copy is effectively backed up every time you run a command.
+  * No commands fail because you have changes in the working copy ("abort: 1
       conflicting file changes: ..."). No need for `sl shelve`.
-    * Simpler and more consistent CLI because the working copy is treated like
+  * Simpler and more consistent CLI because the working copy is treated like
```

Comment:
nit: weird indent in nested list

Reply:
I'm guessing you just mean lines 31 and 33. I could also try indenting the whole nested list more, I don't remember whether that works.